django_loci/apps.py

```diff
@@ -15,11 +25,21 @@ def __setmodels__(self):
         from .models import Location
         self.location_model = Location
 
+    def test_geocoder(self):
+        try:
+            location = geocode('Red Square')
+            assert location is not None
+        except AssertionError:
+            if DJANGO_LOCI_GEOCODE_STRICT_TEST:
+                raise ImproperlyConfigured('Geocode utility is improperly configured')
```

Comment:
this exception needs to show some more detailed information about what is going wrong so the user/developer can fix it